### PR TITLE
fixed repository content listing (issue #33)

### PR DESCRIPTION
**Pull request by emocharnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ use Bitbucket\Client;
 $c = new Client();
 
 $c->authenticate(Client::AUTH_OAUTH_TOKEN, 'your-token-here');
-// $c->authenticate(Client::AUTH_HTTP_PASSWORD, 'your-username', 'your-password');
+// $c->authenticate(Client::AUTH_HTTP_PASSWORD, 'your-username', 'your-passwords');
```
Comment (GrahamCampbell) [inline README.md]: Please revert that change.

File: src/Api/Repositories/Users/Src.php [hunk 2]
```diff
@@ -45,7 +45,7 @@ public function list(array $params = [])
      */
     public function create(array $params = [])
     {
-        $path = $this->buildSrcPath();
+        $path = $this->buildSrcPath(...$params);
```
Comment (GrahamCampbell) [inline src/Api/Repositories/Users/Src.php]: Hmm, I don't think this is quite what we want. What are the exact parameters? And how is `createWithFiles` effected?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Thanks for the PR. I've left some feedback.

File: src/Api/Repositories/Users/Src.php (hunk 2 above)
Comment (emocharnik) [reply src/Api/Repositories/Users/Src.php]: In `createWithFiles` method doesn't pass any parameters into buildSrcPath method. 
Expected behaviour I described in the issue https://github.com/BitbucketAPI/Client/issues/33

Comment (emocharnik) [reply src/Api/Repositories/Users/Src.php]: If you want to list content of specific commit/branch or specific path you have to pass it as query parameter. Here is a request example
`https://api.bitbucket.org/2.0/repositories/atlassian/bbql/src/eefd5ef/tests`

Comment (emocharnik) [reply src/Api/Repositories/Users/Src.php]: sorry, mixed uo the right action method, moved to list

Comment (GrahamCampbell) [reply src/Api/Repositories/Users/Src.php]: That's not a query parameter? `https://example.com/foo?bar=baz`. `bar` is a query parameter in that example, bound to `baz`.

**Comment by GrahamCampbell** [comment]
Oh, I see. You are calling the wrong method. You need to call `show`, not `list`.

**Pull request closed without merging**